src/actions/profileActions.js

```diff
@@ -0,0 +1,147 @@
+import Axios from 'axios';
+import Cookies from 'js-cookie';
+
+export const PROFILE_DETAILS = 'PROFILE DETAILS';
+export const profileDetails = details => (
+  {
+    type: PROFILE_DETAILS,
+    details,
+  }
+);
+
+export const UPDATING = 'updating profile details';
+export const updating = status => (
+  {
+    type: UPDATING,
+    updating: status,
+  }
+);
+
+export const ERRORS = 'failed to update profile details';
+export const errors = errorsDetails => (
+  {
+    type: ERRORS,
+    errors: errorsDetails,
+  }
+);
+
+export const UPDATEPROFILE = 'update profile detail status';
+export const updated = status => (
+  {
+    type: UPDATEPROFILE,
+    status,
+  }
+);
+
+const token = Cookies.get('access_token');
+const username = Cookies.get('username');
+export const getProfileDetails = () => (
+  dispatch => (
+    Axios.get(
+      `https://ah-technocrats.herokuapp.com/api/profiles/${username}/`,
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Token ${token}`,
+        },
+      },
+    ).then(
+      (response) => {
+        dispatch(profileDetails(response.data.profile));
+      },
+      (error) => {
```

Comment:
You could have used `.catch` to catch errors from the backend response. What do you think about this?

Reply:
Did you address this comment?